### PR TITLE
Invite links in the team settings page

### PR DESCRIPTION
**Pull request by jakob223**
cc @keybase/y2ksquad

### CODE REVIEW

File: shared/teams/team/settings-tab/invite-links.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    </Kb.Box2>
+  )
+}
+
+export default InviteLinks
+
+const styles = Styles.styleSheetCreate(
+  () =>
+    ({
+      inviteBox: {
```
Comment (buoyad) [inline shared/teams/team/settings-tab/invite-links.tsx:56]: needs padding

File: shared/teams/team/settings-tab/invite-links.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      {mostRecentInviteLink && (
+        <Kb.Box2 direction="vertical" style={styles.inviteBox}>
+          <Kb.CopyText text={mostRecentInviteLink.url} />
+          <Kb.Text type="BodySmall">
+            Invites as {mostRecentInviteLink.role} · Expires{' '}
+            {formatNicely(mostRecentInviteLink.expirationTime)}
+          </Kb.Text>
+        </Kb.Box2>
+      )}
+      <Kb.Box2 direction="horizontal" gap="tiny" alignSelf="flex-start">
```
Comment (buoyad) [inline shared/teams/team/settings-tab/invite-links.tsx:43]: this is closer to the invite box than designed

File: shared/teams/team/settings-tab/invite-links.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    dispatch(nav.safeNavigateAppendPayload({path: [{props, selected: 'teamInviteHistory'}]}))
+
+  return (
+    <Kb.Box2 direction="vertical" gap="xtiny" fullWidth={true} alignItems="flex-start">
+      <Kb.Text type="BodySmallSemibold">Invite links</Kb.Text>
+      <Kb.Text type="BodySmall">
+        Invite people to the team by sharing {mostRecentInviteLink ? 'this link:' : 'a link.'}
+      </Kb.Text>
+      {mostRecentInviteLink && (
+        <Kb.Box2 direction="vertical" style={styles.inviteBox}>
```
Comment (buoyad) [inline shared/teams/team/settings-tab/invite-links.tsx:35]: this is being centered horizontally in the parent

File: shared/teams/team/settings-tab/invite-links.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+import * as Container from '../../../util/container'
+import * as Types from '../../../constants/types/teams'
+import * as Constants from '../../../constants/teams'
+import * as Styles from '../../../styles'
+import * as dateFns from 'date-fns'
+
+type Props = {
+  teamID: Types.TeamID
+}
+const formatNicely = (etime: number) => (etime === 0 ? 'after 10,000 ys' : dateFns.formatDistanceToNow(etime))
```
Comment (buoyad) [inline shared/teams/team/settings-tab/invite-links.tsx:12]: put in `util/timestamp`. also the designs lack "about". also it says "about 50 years" for me when it actually expires much sooner.

File: shared/teams/team/new-header.tsx [hunk 5]
```diff
@@ -335,7 +336,8 @@ const useHeaderCallbacks = (teamID: TeamID) => {
         )
     : undefined
   const onEdit = nyi
-  const onManageInvites = nyi
+  const onManageInvites = () =>
+    dispatch(nav.safeNavigateAppendPayload({path: [{props: {teamID}, selected: 'teamInviteLinksModal'}]}))
```
Comment (buoyad) [inline shared/teams/team/new-header.tsx:340]: I think it makes more sense for this to go the history modal

File: shared/teams/team/settings-tab/invite-links.tsx (hunk 4 above)
Comment (jakob223) [reply shared/teams/team/settings-tab/invite-links.tsx:12]: Drat, I forgot to multiply by 1000

**Pull request merged**